### PR TITLE
Added tail.md

### LINKED ISSUE #116: tail

### PR DESCRIPTION
**Pull request by nealrp**
# Advanced HW 9 submission

Closes #116 

Name: Neal Parikh
Uniqname: nealrp

### CODE REVIEW

File: _commands/basics/tail.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	}
+		return gcd;
+		*y = x1;
+		*x = y1- (floor(b/a) *x1);
+~~~
+
+##### Break it down
+* Shows the lines of input.txt in reverse order. The default -r option is to display all of the input but this may be changed with the -b, -c, and -n  flags. In this example, we showed the last 5 lines.
+
+#### `man tail`
```
Comment (the-alex) [inline _commands/basics/tail.md]: I get why you would include it, but this last example seems a little unnecessary.

File: _commands/basics/tail.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+		*x = y1- (floor(b/a) *x1);
+		*y = x1;
+		return gcd;
+	}
+	return 1000;
+[waiting for more]
+~~~
+
+##### Break it down
+* Shows the last 5 lines of input.txt. The -f command causes tail to not stop but rather waits for additional data to be added to file
```
Comment (the-alex) [inline _commands/basics/tail.md]: Grammar on line 51: "

Also, nitpick: The -f should be surrounded with backticks- as should every flag.

**Comment by the-alex** [comment]
I don't think there's too much more to add to tail after those changes. I'll merge in once I see them.

**Comment by ppannuto** [comment]
Nice work. Thanks!

If you're interested, I changed the `tail -f` example in bed0f324 to use something that is more likely to generate new data. Also kind of a neat way to spy on what's happening in the background on your machine.

**Pull request merged**